src/index.js

```diff
@@ -98,6 +102,7 @@ app.get("/users/:username/apps/recently-added", recentlyAddedHandler(db));
 app.get("/users/:username/apps/recently-used", recentlyUsedHandler(db));
 app.get("/users/:username/analyses/recent", recentAnalysesHandler());
 app.get("/users/:username/analyses/running", runningAnalysesHandler());
+app.get("/users/:username/apps/popular-featured", popularFeaturedHandler(db));
```

Comment:
I just realized - I added this category to the dashboard endpoint when the user's are logged in.  Not sure if we also want to add it (or only add it) to the endpoint for logged out users. It's a small fix if so, either way.

Reply:
I'm sure we do also want it for logged-out users, as part of showing off what we have to offer, to entice new user sign-ups.